src/main/java/edu/cornell/kfs/fp/batch/ApproveDvsSpawnedByRecurringDvStep.java

```diff
@@ -0,0 +1,26 @@
+package edu.cornell.kfs.fp.batch;
+
+import java.util.Date;
+
+import org.kuali.kfs.sys.batch.AbstractStep;
+
+import edu.cornell.kfs.fp.service.RecurringDisbursementVoucherDocumentService;
+
+public class ApproveDvsSpawnedByRecurringDvStep extends AbstractStep {
```

Comment:
I would have called the batch job AutoApprove, but not need to update the code just for this